src/snowflake/snowpark/_internal/server_connection.py

```diff
@@ -122,6 +122,7 @@ def wrap(*args, **kwargs):
                             func.__name__, duration, sfqid
                         )
                     logger.debug(f"Finished in {duration:.4f} secs")
+                    return result
```

Comment:
added this return so that caller of `upload_stream` gets back the result from put.

https://github.com/snowflakedb/snowpark-python/blob/main/src/snowflake/snowpark/_internal/server_connection.py#L248-L259